loopme-sdk/src/main/java/com/loopme/LoopMeInterstitial.java

```diff
@@ -18,7 +18,6 @@
  * when an ad has been presented or dismissed from the screen, and when an ad has expired or received a tap.
  */
 public final class LoopMeInterstitial extends AdWrapper {
-    public static final String TEST_PORT_INTERSTITIAL = "test_interstitial_p";
```

Comment:
I guess this should be the other way round as the sample-interstitial app is used only in portrait orientation